websocket/frontend/src/todolist/TodoItem.js

```diff
@@ -1,7 +1,40 @@
 import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import { makeStyles } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import DeleteIcon from "@material-ui/icons/Delete";
+
+const useStyles = makeStyles({
+  root: {
+    padding: "2em",
+    height: "3em",
+    margin: "1em"
+  }
+});
 
 export default function TodoItem(props) {
+  const classes = useStyles();
+  function handleDelete() {
```

Comment:
Why put a function in a function?
*always aim to make a function as pure as possible. Easier to test.
Go read up on what `pure functions` mean.